Source/SelfService/Web/application/create.tsx

```diff
@@ -3,140 +3,154 @@
-import { Button } from '../theme/button';
 import { createApplication, HttpApplicationRequest, HttpApplicationEnvironment } from '../api/application';
 import { ShortInfo } from '../api/api';
 import { Guid } from '@dolittle/rudiments';
 
+import { themeDark } from '../theme/theme';
+import { Alert, AlertTitle, Box, Button, Checkbox, FormGroup, FormControlLabel, Link, TextField, Typography } from '@mui/material';
 
 type Props = {};
 
-
 const styles = {
-    root: {
-        width: '100%',
-    },
-    button: {
-        marginTop: 1,
-        marginRight: 1,
-    },
-    actionsContainer: {
-        marginBottom: 2,
+    title: {
+        letterSpacing: '-0.5px',
+        lineHeight: '26px',
     },
-    resetContainer: {
-        padding: 2,
+    secondaryTitle: {
+        textAlign: 'start',
     },
-    inactiveText: {
-        color: '#93959F',
-    },
-    progressBar: {
-        color: '#ff9366',
+    formFieldsWrapper: {
+        display: 'flex',
+        justifyContent: 'space-between',
+        [themeDark.breakpoints.down('sm')]: {
+            flexDirection: 'column',
+            margin: '0',
+        }
     },
-    stepIcon: {
-        'color': '#3B3D48',
-        '&.MuiStepIcon-root.Mui-active': {
-            color: '#6678F6'
+    formField: {
+        'letterSpacing': '0.15px',
+        [themeDark.breakpoints.down('sm')]: {
+            marginBlockEnd: '20px'
         },
-        '&.MuiStepIcon-root.Mui-completed': {
-            color: '#6678F6'
+        '& .MuiInputLabel-root.Mui-error': {
+            'color': themeDark.palette.error.dark,
+            '& .MuiFormLabel-asterisk.Mui-error': {
+                color: themeDark.palette.error.dark,
+            },
         },
-        '&.MuiStepIcon-root.Mui-active .MuiStepIcon-text': {
-            fill: '#B3BBFB'
+        '& .MuiOutlinedInput-root.Mui-error .MuiOutlinedInput-notchedOutline': {
+            color: themeDark.palette.error.dark,
+            borderColor: themeDark.palette.error.dark
         },
-        '&.MuiStepIcon-root .MuiStepIcon-text': {
-            fill: '#93959F'
+        '& .MuiFormHelperText-root.Mui-error': {
+            color: themeDark.palette.error.light,
+            letterSpacing: '0.4px'
         }
+    },
+    actionBtnWrapper: {
+        [themeDark.breakpoints.down('sm')]: {
+            marginBlockStart: '60px'
+        },
+    },
+    actionButtons: {
+        color: themeDark.palette.text.primary,
+        fontSize: '0.875rem',
+        letterSpacing: '0.06em'
+    },
+    alertWrapper: {
+        'marginBlockStart': '48px',
+        'borderColor': themeDark.palette.error.dark,
+        'textAlign': 'left',
+        '& .MuiAlert-icon': {
+            color: themeDark.palette.error.dark,
+        }
+    },
+    supportLink: {
+        color: themeDark.palette.primary.main,
+        textDecoration: 'underline',
+        cursor: 'pointer',
+    }
+};
+
+interface ErrorStates {
+    applicationNameError: {
+        message: String,
```

Comment:
I think the type here should be `string` and not `String`. `String` with a capital `S` is a javascript string type, which can be used to create new strings like this `const someString = new String("This is a string")`. `string` with a small `s` is a typescript string type. You can read more about this [here](https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string).